util/version/Cargo.toml

```diff
@@ -16,9 +16,9 @@ track = "nightly"
 # Latest supported fork blocks.
 # Indicates a critical release in this track (i.e. consensus issue).
 [package.metadata.networks]
-foundation = { forkBlock = 7080000, critical = false }
+foundation = { forkBlock = 7080000, critical = true }
```

Comment:
Shouldn't we revert the `forkBlock` as well to `4370000`?

Reply:
Good catch!